src/studylist/StudyListWithData.js

```diff
@@ -92,6 +92,10 @@ class StudyListWithData extends Component {
       offset: searchData.currentPage * searchData.rowsPerPage,
     };
 
+    if (window.config.supportsFuzzyMatching) {
```

Comment:
You'll want to pull this from the app's `server` config data, not off the window.
The only place `window.config` _should_ be referenced is `App.js` if no config is provided via props.

Reply:
done